plugins/package-managers/node/src/funTest/assets/projects/synthetic/yarn2/dependency-with-patch-expected-output.yml

```diff
@@ -0,0 +1,212 @@
+---
```

Comment:
Commit message:
- "into on*e* of"
- Maybe a ":" after "issues" in the same line

Reply:
> > From how I understand the feature, we should definitely have a follow-up to set `isModified` in such packages.
> 
> yes.

Should I create an issue about this, or do you plan to follow up directly?

Reply:
> Should I create an issue about this

this would be nice, thanks!